apps/src/templates/RenderedMarkdown.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { PropTypes } from 'react';
+import processMarkdown from 'marked';
+import renderer from "../util/StylelessRenderer";
+
+/**
+ * Basic component for rendering a markdown string as HTML.
+ *
+ * Right now, it still uses marked; this will eventually be updated to use the
+ * new remark system, and possibly even support redaction.
+ */
+export default class RenderedMarkdown extends React.Component {
+  static propTypes = {
+    markdown: PropTypes.string.isRequired
+  };
+
+  render() {
+    const processedMarkdown = processMarkdown(this.props.markdown, { renderer });
+    /* eslint-disable react/no-danger */
```

Comment:
I worry a little bit about hiding this warning inside a slick wrapper — consumers may not realize the `markdown` prop is vulnerable to unsafe HTML.  I wonder if we should call this `UnsafeRenderMarkdown` for clarity, until we start limiting what markup can be rendered?